tests/CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ endif()
 
 # must happen after the dll's get copied over
 if(NOT CMAKE_CROSSCOMPILING)
-    gtest_discover_tests(test_regression PROPERTIES DISCOVERY_TIMEOUT 100)
```

Comment:
1000 seconds seems like a long time.

Reply:
Apparently not enough? I guess statically linking the libraries adds to the search time.

```
  Generating Code...
     Creating library D:/a/Vulkan-Loader/Vulkan-Loader/build/tests/Debug/test_regression.lib and object D:/a/Vulkan-Loader/Vulkan-Loader/build/tests/Debug/test_regression.exp
  test_regression.vcxproj -> D:\a\Vulkan-Loader\Vulkan-Loader\build\tests\Debug\test_regression.exe
  CMake Error at C:/Program Files/CMake/share/cmake-3.25/Modules/GoogleTestAddTests.cmake:112 (message):
    Error running test executable.
  
      Path: 'D:/a/Vulkan-Loader/Vulkan-Loader/build/tests/Debug/test_regression.exe'
      Result: Process terminated due to timeout
      Output:
```

Reply:
Let me build this change locally. I have a hunch that it is just hanging and increasing the timeout will do nothing.